code/ui/manager/src/components/mobile/about/MobileAbout.tsx

```diff
@@ -0,0 +1,138 @@
+import type { FC } from 'react';
+import React, { useRef } from 'react';
+import { Transition, type TransitionStatus } from 'react-transition-group';
+import { styled } from '@storybook/theming';
+import { Icons, Link } from '@storybook/components';
+import { UpgradeBlock } from '../../upgrade/UpgradeBlock';
+import { MOBILE_TRANSITION_DURATION } from '../../../constants';
+import { useMobileLayoutContext } from '../MobileLayoutProvider';
+
+export const MobileAbout: FC = () => {
+  const { isMobileAboutOpen, setMobileAboutOpen } = useMobileLayoutContext();
+  const aboutRef = useRef(null);
+  console.log('LOG :', isMobileAboutOpen, setMobileAboutOpen);
+
+  return (
+    <Transition
+      nodeRef={aboutRef}
+      in={isMobileAboutOpen}
+      timeout={MOBILE_TRANSITION_DURATION}
+      appear
+      mountOnEnter
+      unmountOnExit
+    >
+      {(state) => (
+        <Container ref={aboutRef} state={state} transitionDuration={MOBILE_TRANSITION_DURATION}>
+          <Button onClick={() => setMobileAboutOpen(false)} title="Close about section">
+            <Icons icon="arrowleftalt" />
+            Back
+          </Button>
+          <LinkContainer>
+            <LinkLine href="https://github.com/storybookjs/storybook" target="_blank">
+              <LinkLeft>
+                <Icons icon="github" />
+                <span>Github</span>
+              </LinkLeft>
+              <Icons icon="sharealt" width={12} />
+            </LinkLine>
+            <LinkLine
+              href="https://storybook.js.org/docs/react/get-started/install/"
+              target="_blank"
+            >
+              <LinkLeft>
+                <Icons icon="storybook" />
+                <span>Documentation</span>
+              </LinkLeft>
+              <Icons icon="sharealt" width={12} />
+            </LinkLine>
+          </LinkContainer>
+          <UpgradeBlock />
+          <BottomText>
+            Open source software maintained by{' '}
+            <Link href="https://chromatic.com" target="_blank">
+              Chromatic
+            </Link>{' '}
+            and the{' '}
+            <Link href="https://github.com/storybookjs/storybook/graphs/contributors">
+              Storybook Community
+            </Link>
+          </BottomText>
+        </Container>
+      )}
+    </Transition>
+  );
+};
+
+const Container = styled.div<{ state: TransitionStatus; transitionDuration: number }>(
+  ({ theme, state, transitionDuration }) => ({
+    position: 'absolute',
+    width: '100%',
+    height: '100%',
+    top: 0,
+    left: 0,
+    zIndex: 11,
```

Comment:
why do we need to position this absolute?

Reply:
For transitions when the about section arrive on top of the sidebar